Presentation.Web/Models/API/V2/Request/User/CreateUserRequestDTO.cs

```diff
@@ -1,10 +1,19 @@
-﻿using System.Collections.Generic;
+﻿using System.ComponentModel.DataAnnotations;
 
 namespace Presentation.Web.Models.API.V2.Request.User
 {
     public class CreateUserRequestDTO : BaseUserRequestDTO
     {
+        [Required]
+        [EmailAddress]
+        public new string Email { get; set; }
```

Comment:
does this override the other parameters? I'm unfamiliar with this syntax

Reply:
I think if the it receives the object as a CreateUserRequestDTO it will use these 'new' fields, but if it receives it as a BaseUserRequest it will use the base ones.